lib/cf-socket.c

```diff
@@ -871,7 +871,7 @@ static void cf_socket_close(struct Curl_cfilter *cf, struct Curl_easy *data)
       /* this is our local socket, we did never publish it */
```

Comment:
Does that comment still make sense? @icing @bagder

Reply:
I am not sure.